fix(headless): remove createdBy and updateBy for api of headless cms

EmilK15 opened the pull request:
## Related Issue
Closes #828 

## Your solution
The original idea was to figure out how make sure the first/last name appear, but it was later decided to remove the `createdBy` property. Because `createdBy` is referenced by `updatedBy`, I took the liberty of removing references to `updatedBy` since it seemed in the same vein of issue. This change is limited to only `api-headless-cms`.

## How Has This Been Tested?

## Screenshots (if relevant):
I also kept the last modified since this is still relevant to the user.

EmilK15 committed:
fix(headless): remove createdBy and updateBy for api of headless cms

adrians5j reviewed (approved):
Looking good, although I don't know why checks weren't executed :O

adrians5j committed:
chore: remove empty line

A GH action that is validating the PR wasn't triggered for some reason. Hoping this commit will do it.

doitadrian commented:
Not sure why the GH action wasn't triggered here. I made a silly commit, just so the action is triggered and the PR is validated.

Weird, we'll see if this repeats.

doitadrian commented:
@EmilK15 Please check the failing tests. Not sure what's wrong, but we can take a look together if needed.

EmilK15 committed:
fix(headless): add line in category snapshot
